bentoml/cli/deployment.py

```diff
@@ -213,3 +227,225 @@ def check_deployment_status(archive_path, platform, region, stage, api_name):
         deployment.check_status()
 
     return cli
+
+
+def parse_key_value_string(key_value_string):
+    if key_value_string:
+        result = {}
+        for item in key_value_string.split(','):
+            if item is not None:
+                splits = item.split('=')
+                result[splits[0]] = splits[1]
+        return result
+    else:
+        return None
+
+
+def get_deployment_operator_type(platform):
+    return DeploymentOperator.Value(platform.upper())
+
+
+def display_response_status_error(status):
+    _echo('Error code: {}'.format(status.status_code), CLI_COLOR_ERROR)
+    _echo('Error message: {}'.format(status.error_message), CLI_COLOR_ERROR)
+
+
+def display_deployment_info(deployment):
+    _echo('Deployment {} info\n\n'.format(deployment.name))
```

Comment:
use YAML module instead? might be good to add a utility function to convert a Deployment_PB object to JSON or YAML

Reply:
we might need both to_json and from_json, I'm not sure if python protobuf provides that, could you look into it?

Reply:
Good suggestion